js/clinicalTimeline.js

```diff
@@ -218,7 +220,7 @@ window.clinicalTimeline = (function(){
       if (getZoomLevel(minDays, maxDays, width * zoomFactor) !== "days") {
         // add brush overlay
         var xScale = d3.time.scale()
-           .domain([beginning, ending])
+           .domain([minDays, maxDays])
```

Comment:
I think in this case, one should use `beginning` and `ending` instead of `minDays` and `maxDays`. We are determining what traversal over x pixels is expressed as days. This function maps the x axis in days from the first tick (`beginning`) and last tick (`ending`) to the width of the axis in pixels.